src/app/psat/pump-fluid/pump-fluid.component.ts

```diff
@@ -32,6 +32,7 @@ export class PumpFluidComponent implements OnInit {
   inSetup: boolean;
   @Input()
   modificationIndex: number;
+  hasConnectedInventories: boolean;
```

Comment:
hasConnectedInventories value is never set.

Reply:
FYI incase you notice all the other instances of this.. for a flag that I'm not currently using but I'm keeping in as the opinion on the message is displays has flip flopped